WordPress/Classes/ViewRelated/Notifications/Tools/NotificationCommentDetailCoordinator.swift

```diff
@@ -166,21 +72,17 @@ private extension NotificationCommentDetailCoordinator {
             return
         }
 
-        configure(with: notification)
-
-        fetchParentCommentIfNeeded(completion: { [weak self] in
-            self?.refreshViewController()
-        })
+        refreshViewControllerWith(notification)
     }
 
     func showNotificationDetails(with notification: Notification) {
         let storyboard = UIStoryboard(name: Notifications.storyboardName, bundle: nil)
 
         guard let viewController = viewController,
-        let notificationDetailsViewController = storyboard.instantiateViewController(withIdentifier: Notifications.viewControllerName) as? NotificationDetailsViewController else {
-            DDLogError("NotificationCommentDetailCoordinator: missing view controller.")
-            return
-        }
+              let notificationDetailsViewController = storyboard.instantiateViewController(withIdentifier: Notifications.viewControllerName) as? NotificationDetailsViewController else {
+                  DDLogError("NotificationCommentDetailCoordinator: missing view controller.")
+                  return
+              }
```

Comment:
super minor indentation thingy:
```suggestion
        let notificationDetailsViewController = storyboard.instantiateViewController(withIdentifier: Notifications.viewControllerName) as? NotificationDetailsViewController else {
            DDLogError("NotificationCommentDetailCoordinator: missing view controller.")
            return
        }
```

Reply:
Hrm. Xcode indents it my way, not your way. I'm leaving it. I'm not going back and forth because Xcode can't make up it's mind. 😆